server/functions/dolt_procedures.go

```diff
@@ -57,12 +66,43 @@ func initDoltProcedures() {
 	}
 }
 
+// GetDoltStoredProcedure returns the cached ExternalProcedure for the given Dolt procedure name. Returns nil if the
+// procedure is not found.
+func GetDoltStoredProcedure(procedureName string) *plan.ExternalProcedure {
+	return doltExternalProcedures[strings.ToLower(procedureName)]
+}
```

Comment:
We don't need this function. But if we did, this is small enough that it would make more sense to just inline the map search rather than creating an extra function to do that one thing.